CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
+- GitHub Configurations tree item when SWA is connected to local project
+- `Create Configuration File` command to add `staticwebapp.config.json`
+- Standard SKU for `Create Static Web App... (Advanced)`
+
+### Changed
+- `Create Static Web App...` requires an opened workspace in VS Code
+- Location, API location, GitHub org, and repo privacy are defaulted in `Create Static Web App...`
```

Comment:
"defaulted" sounded a bit weird to me, maybe this?

```suggestion
- Location, API location, GitHub org, and repo privacy are automatically set in `Create Static Web App...`
```

Reply:
More wordy, but I feel like using the word default feels kind of important to me 🤔💭
```suggestion
- Location, API location, GitHub org, and repo privacy steps have default values for `Create Static Web App...`
```

Reply:
I dig it 👌